### DISCUSSION TITLE
Is it possible to ignore a specific error?

### DISCUSSION THREAD

**Discussion by hubidubi13** [Hypfer/Valetudo#2122] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but in the process of starting it

### Your questions

Hi!

I just disassembled and flash my used Roborock S7. Everything seemed to go according to plan, I can connect to the robot and start the first mapping run. However, it gets a "Stuck front bumper" error before it actually starts to move. I've done a bit of research, and this seems to be a [common](https://www.reddit.com/r/Roborock/comments/vsn9kj/roborock_s7_error_2_bumper_stuck/) [problem](https://forum.roborock.com/discussion/4781/s7-bumper-stuck-error-2) with the S7. The suggested fixes unfortunately did not help, the issue even arose with the front bumper removed. I then disassembled the Robot again to verify all cables are connected, but I cannot see anything wrong in comparison with the photos I took during the first disassembly.

My question is, if I can simply ignore this error somehow? The front bumper is obviously not stuck and works as it should. I've done a short text search in the code and found the bit where the message for the webUI is generated. But I am assuming changing the severity level and kind there will only affect the webUI and not actually the behavior of the robot?

Thank you anyways for this amazing piece of software! I will definitely continue trying to get it to work!

**Comment by Hypfer** [comment, marked as answer]
No, you need to fix it
